Core/MIPS/ARM64/Arm64RegCacheFPU.cpp

```diff
@@ -319,6 +319,7 @@ void Arm64RegCacheFPU::FlushR(MIPSReg r) {
 		if (mr[r].reg == INVALID_REG) {
 			ERROR_LOG(JIT, "FlushR: MipsReg had bad ArmReg");
 		}
+		FlushArmReg((ARM64Reg)(S0 + mr[r].reg));
```

Comment:
This Arm64 change maybe belongs to a different PR? Though not saying it looks wrong. Did you notice a problem?

Reply:
Well, nothing is actually using FlushR() or FlushV() in arm64jit currently, so this actually changes nothing.  But FlushR() was accidentally a DiscardR() before, because it didn't actually flush it, so I figured I'd fix it in case we ever use it...

-[Unknown]